src/js/components/returns/inbound/AddItemsPage.jsx

```diff
@@ -260,14 +260,9 @@ class AddItemsPage extends Component {
       apiClient.get(url)
         .then((resp) => {
           const inboundReturn = parseResponse(resp.data.data);
-          const returnItems = inboundReturn.stockTransferItems.length > 0 ?
-            _.map(inboundReturn.stockTransferItems, item => ({
-              ...item,
-              product: {
-                ...item.product,
-                label: `${item.product.productCode} ${item.product.translatedName ?? item.product.name}`,
-              },
-            })) : new Array(1).fill({ sortOrder: 100 });
+          const returnItems = inboundReturn.stockTransferItems.length > 0
```

Comment:
I know it's not your code, but if we already "touch" some part of the code that could be simplified, just a short suggestion, which I'm sure you are aware of, but just didn't see it
```suggestion
          const returnItems = inboundReturn.stockTransferItems.length
```